sbom.cdx.json

```diff
@@ -0,0 +1,72 @@
+{
+  "bomFormat": "CycloneDX",
+  "specVersion": "1.6",
+  "version": 1,
+  "metadata": {
+    "authors": [
+      {
+        "name": "@VCS_SBOM_AUTHORS@"
+      }
+    ]
+  },
+  "components": [
+    {
+      "type": "firmware",
+      "bom-ref": "pkg:github/tianocore/edk2@@VCS_TAG@",
+      "cpe": "cpe:2.3:a:tianocore:edk2:@VCS_TAG@:*:*:*:*:*:*:*",
+      "name": "EDK II",
+      "version": "@VCS_VERSION@",
+      "description": "A cross-platform firmware development environment for UEFI and PI specifications",
+      "authors": [
+        {
+          "name": "@VCS_AUTHORS@"
+        }
+      ],
+      "supplier": {
+        "name": "EDK II developers"
+      },
+      "licenses": [
+        {
+          "license": {
+            "id": "Apache-2.0"
+          }
+        },
+        {
+          "license": {
+            "id": "BSD-2-Clause"
```

Comment:
EDK2 uses the  license `BSD-2-Clause-Patent`. I'm not sure if that's a known license type in this file though?

Reply:
Yup, that's fine, fixed -- good catch.